admin/server/organizations.go

```diff
@@ -274,6 +281,11 @@ func (s *Server) AddOrganizationMember(ctx context.Context, req *adminv1.AddOrga
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	err = s.admin.NotifyUserAdditionToOrganization(ctx, req.Email, org.Name, role.Name)
+	if err != nil {
+		return nil, status.Error(codes.Internal, err.Error())
+	}
```

Comment:
This feels a little redundant. Let's just export the emailer, so it becomes `s.admin.Email...`.

Ideally, we would move the whole thing into a `s.admin.AddOrgMember` func, but don't have to refactor like that now.